src/enums/viewComponentNames.ts

```diff
@@ -5,11 +5,19 @@
 export enum ViewComponentNames {
   SIGN_IN = 'signin',
   SIGN_OUT = 'signout',
+
+  // Incorporation View Components
   ADD_PEOPLE_AND_ROLES = 'add-people-and-roles',
   CREATE_MEMORANDUM = 'create-memorandum',
   CREATE_RULES = 'create-rules',
   CREATE_SHARE_STRUCTURE = 'create-share-structure',
   DEFINE_COMPANY = 'define-company',
   INCORPORATION_AGREEMENT = 'incorporation-agreement',
-  REVIEW_CONFIRM = 'review-confirm'
+  REVIEW_CONFIRM = 'review-confirm',
+
+  // Dissolution View Components
+  DEFINE_DISSOLUTION = 'define-dissolution',
+  UPLOAD_SPECIAL_RESOLUTION = 'upload-special-resolution',
+  UPLOAD_AFFIDAVIT = 'upload-affidavit',
+  REVIEW_CONFIRM_DISSOLUTION = 'review-confirm-dissolution'
 }
```

Comment:
This looks like the same list as Route Names. Could we have just a single list? Perhaps with a comment somewhere explaining that the route names are the same as the component names.

Reply:
This came up before, it was your recommendation we have separate enums despite the values being the same, as you didn't like the idea of using the same enum for separate properties in the routes, you are keeping me on my toes here haha

Reply:
![Screen Shot 2021-09-30 at 2 21 27 PM](https://user-images.githubusercontent.com/53542131/135532005-6d7bfec9-9806-4cec-9820-0d25348ad616.png)

Reply:
Haha, ok, nvm.